src/components/About/About.module.css

```diff
@@ -9,59 +10,238 @@
 }
 
 .avatar {
-  width: 200px;
+  width: 175px;
   height: 200px;
+  border-radius: 9px;
 }
 
 .user {
   display: flex;
-  justify-content: space-evenly;
-  padding-top: 10px;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10px 10px;
 }
 
 .user__wrap {
-  line-height: .8;
-  padding-top: 40px;
+  margin-bottom: 14px;
+}
+
+.user__info {
+  line-height: 0;
+  padding-left: 32px;
+  padding-bottom: 30px;
 }
 
 .user__name {
-  font-size: 23px;
+  font-size: 28px;
+  color: #000;
 }
 
-.user__info {
-  font-size: 17px;
+.user__bio {
+  color: #999;
+  font-weight: normal;
+  font-size: 12px;
 }
 
 .list {
-  padding-right: 40px;
+  padding: 20px 32px 0px 32px;
 }
 
 .repo {
   text-decoration: none;
-  color: #ff4081;
-  font-size: 20px;
+  color: #000;
   font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
   display: flex;
-  justify-content: center;
+  justify-content: flex-start;
+  padding-top: 7px;
+  padding-left: 19px;
 }
 
 .item {
-  border: 1px solid #ff4081;
+  width: 636px;
+  height: 54px;
+  border: 1px solid #ccc;
   border-radius: 10px;
   box-sizing: border-box;
   margin: 0 0 10px 0;
-  padding: 5px;
 }
 
-.item:hover {
-  background-color: #ff4081;
+.item:hover .repo {
+  color: #356eff;
 }
 
-.item:hover .repo {
+.info__contacts {
+  padding-top: 35px;
+}
+
+.mail__href,
+.tg__href {
+  text-decoration: none;
+  color: #999;
+  font-size: 12px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 9px
+}
+
+.mail,
+.phone {
+  padding-left: 4px;
+  font-weight: normal;
+  font-size: 12px;    
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5830965)